tests/test_notifications.py

```
@@ -0,0 +1,45 @@
+            mock_open.return_value.__enter__.return_value = object()
+            success, error = NotificationService.send_ntfy(
+                'https://ntfy.example.com/topic', 'Title', 'Message', token='tk_secret'
+            )
+        assert success is True
+        req = mock_open.call_args[0][0]
+        assert req.get_header('Authorization') == 'Bearer tk_secret'
+
+    def test_bare_topic_gets_no_header_without_token(self, app):
+        with patch('app.services.notifications.urlopen') as mock_open:
+            mock_open.return_value.__enter__.return_value = object()
+            success, error = NotificationService.send_ntfy(
```

Ruff (0.16.0)

[warning] Unpacked variable `error` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)